substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/Target_jdk_jfr_internal_Repository.java

```diff
@@ -24,22 +24,46 @@
  */
 package com.oracle.svm.core.jfr;
 
+import java.io.IOException;
+import java.nio.file.Path;
+
 import com.oracle.svm.core.SubstrateUtil;
 import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.RecomputeFieldValue;
 import com.oracle.svm.core.annotate.Substitute;
 import com.oracle.svm.core.annotate.TargetClass;
+import com.oracle.svm.core.annotate.TargetElement;
 
-import jdk.jfr.internal.SecuritySupport.SafePath;
+import jdk.jfr.internal.SecuritySupport;
```

Comment:
I think there's a few places like here where `import jdk.jfr.internal.SecuritySupport;` is no longer needed.